tests/common_instancetypes_test.go

```diff
@@ -37,12 +37,14 @@ var _ = Describe("Common Instance Types", func() {
 			}
 		})
 		It("should reconcile from URL when provided", func() {
-			URL := "https://github.com/kubevirt/common-instancetypes//VirtualMachineClusterPreferences?ref=v0.1.0"
+			URL := "https://github.com/kubevirt/common-instancetypes//VirtualMachineClusterPreferences?ref=v0.3.3"
```

Comment:
I know isn't a good practice to set main branches as a reference, but can we do it? instead of bumping this tag. @0xFelix What did you mean when you said the URL is wrong, did you mean to the ref version?

Reply:
No, this isn't set to `main` as we'd then need another change each time a release branch is cut pinning said branch to a known supported release of common-instancetypes by SSP and the underlying KubeVirt version we support.

This is all being deprecated in `v0.19.0` FWIW so I wouldn't get too invested in this.

Reply:
Ack, thanks.

Reply:
I meant the URL is only referring to `VirtualMachineClusterPreferences`, but not `VirtualMachineClusterInstancetypes`, while below we also test that all expected got deployed `VirtualMachineClusterInstancetypes` (although 0 of 0 is also a correct assertion).